course.py

```diff
@@ -248,3 +248,44 @@ def add_duplicate_course():
         db.session.rollback()
 
     return redirect(url_for('course.course_info', course_id=course_id, current_year=year))
+
+
+@course_bp.route('/evaluations')
+@login_required
+def evaluations():
+    user_id = request.args.get('user_id')
+    current_year = request.args.get('current_year')
+    courses = db.session.query(Course).filter_by(year=current_year).all()
+
+    return render_template('evaluations.html', courses=courses, current_year=current_year, user_id=user_id)
+
+
+@course_bp.route('/create_evaluations', methods=['POST'])
+@login_required
+def create_evaluation():
+    form = request.form
+    if not form:
+        return make_response("Problem with form request", 500)
+
+    course_id = request.form['course_id']
```

Comment:
Beware, you may get a malformed form and not get the expected keys. Please handle this case.